lib/matplotlib/legend.py

```diff
@@ -208,8 +209,11 @@ def _update_bbox_to_anchor(self, loc_in_canvas):
     Whether round edges should be enabled around the `~.FancyBboxPatch` which
     makes up the legend's background.
 
-shadow : bool, default: :rc:`legend.shadow`
+shadow : bool or color, default: :rc:`legend.shadow`
     Whether to draw a shadow behind the legend.
+    If value is a color, a shadow of that color will be applied.
+    If the value is neither boolean nor a valid color, behavior is that of
+    *True* and the shadow is the default color.
```

Comment:
This is no longer allowed, no?

Reply:
Multiple-type flags? See discussion above with @jklymak and @anntzer.

Reply:
I mean this:
>      If the value is neither boolean nor a valid color

Reply:
Oh, I'm sorry. I'll fix that.